src/node/internal/internal_tls_common.ts

```diff
@@ -56,12 +55,16 @@ export function SecureContext(
       maxVersion
     );
   }
-  if (minVersion !== undefined) {
-    throw new ERR_OPTION_NOT_IMPLEMENTED('minVersion');
-  }
-  if (maxVersion !== undefined) {
-    throw new ERR_OPTION_NOT_IMPLEMENTED('maxVersion');
-  }
+  // TODO(soon): Support this.
```

Comment:
I don't think "Support this" is a good todo here. We don't have any way of telling the underlying layers which TLS version to support. I would probably just replace this entire commented block with something like:

```js
// We do not support the minVersion and maxVersion options at this
// time and will just ignore them if they are passed.
```